### ISSUE TITLE
Differences between v1 and v2 re: commanline arguments

### ISSUE THREAD

**Issue by aptalca** [cdr/code-server#1019]
Hi, 

Looking at the output of `code-server --help`, I see there are a lot of differences between v1 and v2. Is there any documentation that details the changes for users upgrading from v1 to v2?

More specifically. . .
1) Is `-d` still supported or only `--user-data-dir`?
2) Is `-e` still supported or only `--extensions-dir`?
3) Is `--disable-telemetry` still supported?
4) Is `-N` supported or is it automatic based on whether the `PASSWORD` env var is set?
5) Is `-H` supported or is it automatic based on whether a cert is selected?

Also I see that the documentation was updated to reference port `8080` instead of `8443` everywhere, but the latest stable release still uses `8443`, just fyi

Thanks

**Comment by sr229** [comment]
Please refer to the `--help` output. 

TLDR: we only pass VSCode commands going forward with our additions. See GH-857.

**Comment by sr229** [comment]
As for the port change, it was requested for our venerable Google Cloud Run users.

**Comment by aptalca** [comment]
I guess I didn't make myself clear. My questions above were a result of comparing the `--help` output in the two versions.

I'll go ahead and assume the `--help` output of v2 is 100% accurate and that all those options from v1 have been dropped. 

Then how do I make sure telemetry is disabled? I don't see a telemetry entry in the code-server gui settings as described in vscode docs: https://code.visualstudio.com/docs/getstarted/telemetry#_disable-telemetry-reporting

Thanks

**Comment by sr229** [comment]
@aptalca you can use the `--disable-telemetry` option. VSCode web by default sends telemetry and there is no GUI to support such on upstream.

**Comment by aptalca** [comment]
Thanks. 

Then perhaps that option should be added to the `--help` output?

**Comment by antofthy** [comment]
--disable-telemetry is not listed in the --help for v2.

Instead --telemetry is listed however any attempt my me to use it results in "error File not found"

**Comment by code-asher** [comment]
We just display what VS Code displays and it seems they opt not to show `--disable-telemetry` but it's still a valid flag. https://github.com/cdr/code-server#telemetry. The options that do appear are accurate and the ones that no longer appear are gone with the exception of `--disable-telemetry`.

I'm not sure what `--telemetry` is supposed to do but when I run it it seems to require a `telemetry-core.json` file and fails when it can't find it (the same thing happens to me with "normal" VS Code).

I'm not sure why the disable telemetry option doesn't show in the settings but it's on my list to look into.
